app/src/main/java/me/ccrama/redditslide/Activities/MainActivity.java

```diff
@@ -890,7 +896,7 @@ public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which)
             @Override
             public void onClick(View v) {
                 Intent i = new Intent(MainActivity.this, Settings.class);
-                startActivityForResult(i, 1);
+                startActivityForResult(i, SETTINGS_RESULT);
```

Comment:
Shouldn't it be RESET_THEME_RESULT? Exiting the settings reloaded the theme before.